hack/e2e-test.sh

```diff
@@ -0,0 +1,28 @@
+export KUSTOMIZE=$PWD/bin/kustomize
+export GINKGO=$PWD/bin/ginkgo
+export KIND=$PWD/bin/kind
+function cleanup {
+    if [ $USE_EXISTING_KIND == 'false' ] 
+    then 
+        $KIND delete cluster --name $KIND_CLUSTER_NAME
+    fi
+}
+function startup {
+    if [ $USE_EXISTING_KIND == 'false' ] 
+    then 
+        $KIND create cluster --name $KIND_CLUSTER_NAME --image $E2E_KIND_VERSION
+    fi
+}
+function kind_load {
+    $KIND load docker-image $IMAGE_TAG --name $KIND_CLUSTER_NAME
+}
+function kueue_deploy {
+    cd config/components/manager && $KUSTOMIZE edit set image controller=$IMAGE_TAG
```

Comment:
```suggestion
    (cd config/components/manager && $KUSTOMIZE edit set image controller=$IMAGE_TAG)
```

then you don't need to do `cd ../../../` below

Reply:
Done.